app/src/main/res/layout-sw600dp-land/profile_chooser_fragment.xml

```diff
@@ -1,125 +0,0 @@
-      android:overScrollMode="never"
-      android:scrollbars="none">
-
-      <androidx.constraintlayout.widget.ConstraintLayout
-        android:id="@+id/profile_chooser_container"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent">
-
-        <ImageView
-          android:id="@+id/profile_chooser_language_icon"
-          android:layout_width="48dp"
-          android:layout_height="48dp"
-          android:layout_marginEnd="36dp"
-          android:paddingStart="4dp"
-          android:paddingTop="20dp"
-          android:paddingEnd="20dp"
-          app:srcCompat="@drawable/ic_language_icon_grey_24dp"
-          android:visibility="gone"
-          app:layout_constraintEnd_toEndOf="parent"
-          app:layout_constraintTop_toTopOf="parent" />
-
-        <TextView
-          style="@style/Subtitle1ViewCenter"
-          android:minHeight="48dp"
-          android:paddingTop="20dp"
-          android:text="@string/profile_chooser_language"
-          android:textColor="@color/profile_chooser_grey_text"
-          android:visibility="gone"
-          app:layout_constraintEnd_toStartOf="@+id/profile_chooser_language_icon"
-          app:layout_constraintTop_toTopOf="parent" />
-
-        <TextView
-          android:id="@+id/profile_select_text"
-          android:layout_width="wrap_content"
-          android:layout_height="wrap_content"
-          android:layout_marginStart="96dp"
-          android:layout_marginTop="80dp"
-          android:fontFamily="sans-serif"
-          android:text="@string/profile_chooser_select"
-          android:textColor="@color/color_def_white"
-          android:textSize="36sp"
-          app:layout_constraintStart_toStartOf="parent"
-          app:layout_constraintTop_toTopOf="parent" />
-
-        <androidx.recyclerview.widget.RecyclerView
-          android:id="@+id/profile_recycler_view"
-          android:layout_width="match_parent"
-          android:layout_height="match_parent"
-          android:clipToPadding="false"
-          android:fadingEdge="horizontal"
-          android:fadingEdgeLength="72dp"
-          android:orientation="vertical"
-          android:overScrollMode="never"
-          android:paddingBottom="@{hasProfileEverBeenAddedValue ? @dimen/profile_chooser_padding_bottom_profile_already_added: @dimen/bottom_white_space}"
-          android:requiresFadingEdge="vertical"
-          android:scrollbars="none"
-          android:tag="profile_recycler_view"
-          app:data="@{viewModel.profiles}"
-          app:layoutMarginEnd="@{hasProfileEverBeenAddedValue ? @dimen/profile_chooser_margin_end_profile_already_added : @dimen/space_0dp}"
-          app:layoutMarginStart="@{hasProfileEverBeenAddedValue ? @dimen/profile_chooser_margin_start_profile_already_added : @dimen/space_0dp}"
-          app:layoutMarginTop="@{hasProfileEverBeenAddedValue ? @dimen/space_0dp : @dimen/profile_chooser_margin_top_profile_not_added}"
```

Comment:
This part is inconsistent with the tablet portrait version--it seems like it will be a regression per the diff. Is the new result correct?